test/end2end_test.go

```diff
@@ -5012,12 +5016,15 @@ func (ss *stubServer) FullDuplexCall(stream testpb.TestService_FullDuplexCallSer
 
 // Start starts the server and creates a client connected to it.
 func (ss *stubServer) Start(sopts []grpc.ServerOption, dopts ...grpc.DialOption) error {
-	r := manual.NewBuilderWithScheme("whatever")
-	ss.r = r
+	if ss.network == "" && ss.address == "" && ss.target == "" {
```

Comment:
This could lead to an error case (if one of them is non empty).

I think we can add a bool `unix` instead.
If `unix == true`, make a temp file as the address (and you don't need address or target from the input).
If `unix == false`, do the old thing.

Reply:
It seems we need `target`, though, to test the 3 forms of the target string.

I actually don't mind this idea.  But maybe:
```go
if ss.network == "" { ss.network = "tcp" }
if ss.addr == "" { ss.address = "localhost:0" }
if ss.target == "" {
  ss.r = manual.NewBuilderWithScheme("whatever")
  ss.target = ss.r.Scheme() + ":///" + ss.addr
}
```